client/lib/posts/test/post-edit-store.js

```diff
@@ -18,8 +17,8 @@ describe( 'post-edit-store', function() {
 	useFakeDom();
 
 	beforeEach( function() {
-		PostEditStore = rewire( '../post-edit-store' );
-		dispatcherCallback = PostEditStore.__get__( 'dispatcherCallback' );
+		PostEditStore = require( '../post-edit-store' );
+		dispatcherCallback = PostEditStore.dispatcherCallback;
```

Comment:
Rather than exposing the callback, could we use a similar technique to the one above in post formats store in [spying the `Dispatcher.register` calls](https://github.com/Automattic/wp-calypso/blob/5399c925d0c564584532a38f90f5cad23be8a144/client/lib/post-formats/test/store.js#L22-L24)?